src/order/filter/at_top_bot.lean

```diff
@@ -602,7 +602,7 @@ end
 
 /-- If `f` is a monotone sequence of `finset`s and each `x` belongs to one of `f n`, then
 `tendsto f at_top at_top`. -/
-lemma monotone.tendsto_at_top_finset [preorder β]
+lemma tendsto_at_top_finset_of_monotone [preorder β]
```

Comment:
why did you change this name?

Reply:
This code is inside `namespace filter`, so I used a non-dot-style name with a dot-style `alias` to escape the namespace.